web-frontend/src/main.tsx

```diff
@@ -41,12 +42,12 @@ const router = createBrowserRouter(
       <Route path="account-create-step-two" element={<StepTwo />} />
       <Route path="account-create-step-three" element={<StepThree />} />
       <Route path="home" element={<Home />} />
-      <Route path="transactions" element={<></>}></Route>
-      <Route path="send-coins" element={<SendCoins />}></Route>
-      <Route path="receive" element={<></>}></Route>
-      <Route path="assets" element={<></>}></Route>
-      <Route path="settings" element={<></>}></Route>
-      <Route path="contacts" element={<></>}></Route>
+      <Route path="transactions" element={<></>} />
```

Comment:
`element={<></>}` !? 🤔  Is that temporary because we haven't finish the pages!? Am i right!?

Reply:
yes

Reply:
@mazmassa Yes!